src/actors/runtime/AbstractActor.ts

```diff
@@ -54,10 +54,10 @@ export default abstract class AbstractActor {
   constructor(daprClient: DaprClient, id: ActorId) {
     this.daprClient = daprClient;
     this.actorClient = new ActorClient(
-      daprClient.options.daprHost,
-      daprClient.options.daprPort,
-      daprClient.options.communicationProtocol,
-      daprClient.options,
+      daprClient.daprClient.options.daprHost,
+      daprClient.daprClient.options.daprPort,
+      daprClient.daprClient.options.communicationProtocol,
+      daprClient.daprClient.options,
```

Comment:
Could you please remind me why we are not using `daprClient.options` directly?

Reply:
It was because options on DaprClient were changed to partial, but I solved the issue in another way and changed it back.